test/fuzz/p2p/addrbook/fuzz.go

```diff
@@ -0,0 +1,38 @@
+package addr
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"net"
+
+	"github.com/tendermint/tendermint/p2p"
+	"github.com/tendermint/tendermint/p2p/pex"
+)
+
+var addrBook = pex.NewAddrBook("./testdata/addrbook.json", true)
```

Comment:
Any sort of cleanup needed for this?

Reply:
Nope (because GH actions are executed in Docker containers? all modifications to addrbook.json will be discarded after running the test)